Library/Homebrew/cmd/test-bot.rb

```diff
@@ -427,7 +427,7 @@ def formula(formula_name)
         "#{@tap}/#{formula_name}"
       end
 
-      test "brew", "uses", "--recursive", canonical_formula_name
+      test "brew", "uses", canonical_formula_name
```

Comment:
Good point. I guess we should keep the logic for time being. And manually check all bottles for missing deps. Then to remove it.